src/components/styles/Dishes.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+@import url('https://fonts.cdnfonts.com/css/lato');
+
+@media only screen and (max-width: 768px) {
+  .wrapper {
+    background-color: #2e5cb8;
+    height: 100%;
+  }
+
+  .dish-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin: 2rem 1rem 1rem;
+    background-color: #2e5cb8;
+    height: 150vh;
+  }
+
+  .cookpad-title {
+    text-align: center;
+    color: #0ff;
+    padding-top: 12px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .search-bar {
+    background-color: #fff;
+    margin: 0 30px;
+    border-radius: 5px;
+    width: auto;
+  }
+
```

Comment:
- [ ] [OPTIONAL] Good job so far 👏 In addition to what you have done, I suggest removing the margin at the top of the home page and details page. Let me show it to you in the screenshot below 👇 

![image](https://user-images.githubusercontent.com/90222110/234990554-9225fd02-d64d-4344-b6b6-79243931c083.png)

Reply:
@BertrandConxy Thanks a lot for the feedback. cookpad-title, margin: 0 fixed the problem.